pyls_mypy/plugin.py

```diff
@@ -41,17 +44,45 @@ def parse_line(line, document=None):
 
 @hookimpl
 def pyls_lint(config, document):
+    args = ['--incremental',
+            '--show-column-numbers',
+            '--follow-imports', 'silent']
+
+    config_files = config.plugin_settings('pyls_mypy').get('config_files', [])
+    config_files = (os.path.expandvars(os.path.expanduser(p.strip()))
+                    for p in config_files)
+    config_files = [p for p in config_files if p]
+    while config_files:
+        # This seems a little complicated but the idea is that configuration
+        # files in relative paths that are not separated by absolute path
+        # configuration files, will all have an equal chance of being found
+        # in each directory before the search moves up the project tree.
+        config_file = None
+        relative = list(itertools.takewhile(lambda p: not os.path.isabs(p),
+                                            config_files))
+        if relative:
+            config_files = config_files[len(relative):]
+
+            found = config.find_parents(os.getcwd(), relative)
```

Comment:
I think it should be `paths` or something similar. It not immediately obvious what this pyls-specific function does, so naming would have helped a reader.